android/library/src/main/java/com/liferay/mobile/screens/base/BaseScreenlet.java

```diff
@@ -120,7 +130,17 @@ protected I prepareInteractor(String actionName) {
 	protected void init(Context context, AttributeSet attributes) {
 		LiferayScreensContext.init(context);
 
-		_screenletView = createScreenletView(context, attributes);
+		assignView(createScreenletView(context, attributes));
+	}
+
+	public void render(int layoutId) {
```

Comment:
No, it's just for the use case where you want to dynamically create a screenlet from code.